src/js/port_handler.js

```diff
@@ -100,7 +100,6 @@ PortHandler.removePort = function(currentPorts) {
     const removePorts = self.array_difference(self.initialPorts, currentPorts);
 
     if (removePorts.length) {
-        console.log(`PortHandler - Removed: ${JSON.stringify(removePorts)}`);
```

Comment:
Not sure why this was removed?

Reply:
Left from previous iteration. Restored for debugging.